### ISSUE TITLE
Container doesn't work anymore

### ISSUE THREAD

**Issue by morsik** [hamburml/docker-flow-letsencrypt#19]
Updated few minutes ago, and it just doesn't works:

```
root@myhost ~ # docker logs -f c99d0d2d040a
Starting Docker Flow: Let's Encrypt
Docker Flow: Let's Encrypt started
We will use mymail@ishere.com for certificate registration with certbot. This e-mail is used by Let's Encrypt when you lose the account and want to get it back.
usage: 
  certbot-auto [SUBCOMMAND] [options] [-d DOMAIN] [-d DOMAIN] ...

Certbot can obtain and install HTTPS/TLS/SSL certificates.  By default,
it will attempt to use a webserver both for obtaining and installing the
cert. 
certbot: error: unrecognized arguments: 
Unable to verfiy domain ownership, we try again in 5 seconds.
usage: 
  certbot-auto [SUBCOMMAND] [options] [-d DOMAIN] [-d DOMAIN] ...

Certbot can obtain and install HTTPS/TLS/SSL certificates.  By default,
it will attempt to use a webserver both for obtaining and installing the
cert. 
certbot: error: unrecognized arguments: 
Unable to verfiy domain ownership, we try again in 5 seconds.
usage: 
  certbot-auto [SUBCOMMAND] [options] [-d DOMAIN] [-d DOMAIN] ...

Certbot can obtain and install HTTPS/TLS/SSL certificates.  By default,
it will attempt to use a webserver both for obtaining and installing the
cert. 
certbot: error: unrecognized arguments: 
Unable to verfiy domain ownership, we try again in 5 seconds.
usage: 
  certbot-auto [SUBCOMMAND] [options] [-d DOMAIN] [-d DOMAIN] ...

Certbot can obtain and install HTTPS/TLS/SSL certificates.  By default,
it will attempt to use a webserver both for obtaining and installing the
cert. 
```

After one of those commits:
- https://github.com/hamburml/docker-flow-letsencrypt/commit/737c196f14ab69f67ac7dda50d36bc3cc05ec7c7
- https://github.com/hamburml/docker-flow-letsencrypt/commit/881320f87c414797595fd466d582db1d0710623d

**Comment by hamburml** [comment]
Are you using latest-tag?

If so please switch to v0.1.2 or v.0.1.1. https://store.docker.com/community/images/hamburml/docker-flow-letsencrypt/tags

Looks like the dry-run mechanism isn't working correctly but I haven't had time to test it myself. Only latest-Tag has the dry-run PR merged, please switch to release versions. Will check this asap, sorry for any inconvenience caused.

*edit*

I was able to verify that this happens when using latest, v0.1.2 is still working on my end. Will be fixed. Thanks for the report!

**Comment by morsik** [comment]
Thanks you very much. Now it works better ;)

But it could be nice to mention about possible tags in README, cause I was reading it 3 times to check if something has changed, and I was even looked at that `:latest` in tag name ;)

**Comment by jorianvo** [comment]
@hamburml can you share what you did to verify, so I can take a look at it?

**Comment by hamburml** [comment]
@jorianvo I just updated version from v0.1.2 to latest and it wasn't working anymore. 

> certbot: error: unrecognized arguments: 

Is the error, so it looks some of the arguments are wrong.

@morsik You are right, I will edit the README when it's fixed.

**Comment by jorianvo** [comment]
I'm sorry for any inconvenience I may have caused and that it took longer to fix than I hoped (I've been very busy but I just created the pull request with the fix.

**Comment by morsik** [comment]
This is still broken because of wrongly merged code which sets `args` variable in `certbot.sh` twice. And second time wrongly with `"$staging"` which makes it unusable, cause it's passed as empty argument to `certbot` and it stopped working.

I fixed this in #21 PR, so I'm closing this in favor of #21 PR and #3 Issue.
